Loop/Managers/Alerts/StoredAlert.swift

```diff
@@ -162,6 +174,29 @@ extension Alert.Trigger {
             } else {
                 throw StorageError.invalidStoredInterval
             }
+        case 3:
+            if let storedDateMatching = storedDateMatching {
+                if let storageDate = storageDate,
+                   let nextDate = Calendar.current.nextDate(after: storageDate, matching: storedDateMatching, matchingPolicy: .nextTime),
+                   // Interesting case here, and I'm not exactly sure what to do.
+                   // If the "next matching date" after storage date is in the past, that means we've past the time when the alert should have shown
+                    // So... make it .immediate?? (TODO: Or, maybe we should throw an error?  Not clear...)
```

Comment:
[nit] TODO?  Immediate seems reasonable to me considering that is the same behavior for a delayed alert past its time. Either way, if it is an actual TODO (as in to be determined) then please create a Jira ticket to capture this decision/work.

Reply:
However, I suppose there is also the question of repeating or nextDateRepeating in that what if it is past the time an alert should have been shown (regardless of whether it will be repeated and shown again in the future), yes? For the repeating case, won't we miss an alert here?

Reply:
I don't quite follow.  The purpose of this logic is an optimization to delete alerts from the `AlertStore` that were issued but they have not yet been presented.  A repeating alert (either an interval-based one or a date-matching one) that past it's first presentation time should not be deleted; we should record the retraction as normal.  I think this is correct.